app/controllers/api/v1/invoices_controller.rb

```diff
@@ -281,14 +291,12 @@ def preview_params
             :tax_identification_number,
             :currency,
             :timezone,
-            shipping_address: [
-              :address_line1,
-              :address_line2,
-              :city,
-              :zipcode,
-              :state,
-              :country
-            ],
+            :address_line1,
+            :address_line2,
+            :city,
+            :zipcode,
+            :state,
+            :country,
```

Comment:
As far as I understand, `address_line1` and `shipping_address_line1` are 2 separate attributes, so her we're not "removing the nesting", we change from shipping address to main address. Correct?